source/includes/options-mongotranslate.yaml

```diff
@@ -0,0 +1,119 @@
+program: mongotranslate
+name: query
+directive: option
+description: |
+  Specifies a SQL query to translate into a MongoDB aggregation
+  pipeline. Either ``--query`` or :option:`--queryFile
+  <mongotranslate --queryFile>` is required.
+---
+program: mongotranslate
+name: queryFile
+directive: option
+description: |
+  Specifies a path to a file containing a SQL query to translate
+  into a MongoDB aggregation pipeline. Either :option:`--query
+  <mongotranslate --query>` or ``--queryFile`` is required.
+---
+program: mongotranslate
+name: schema
+directive: option
+description: |
+  *Required.* Specifies a ``.drdl`` schema file or a directory
+  containing one or more ``.drdl`` schema files created by the
+  :binary:`~bin.mongodrdl` program to use when translating a
+  SQL query into an aggregation pipeline.
+---
+program: mongotranslate
+name: format
+directive: option
+default: |
+  ``multiline``
+description: |
+  *Optional.* Valid options are ``none`` and ``multiline``. The
+  default ``multiline`` option displays results in a more easily
+  readable format.
+  
+  .. list-table::
+     :header-rows: 1
+     :widths: 30 30 40
+
+     * - Command Type
+       - Option
+       - Description
+
+     * - Without the :option:`--explain <mongotranslate --explain>` option 
+       - ``--format none``
+       - Returns an aggregation pipeline all on one line.
+
+     * - Without the :option:`--explain <mongotranslate --explain>` option
+       - ``--format multiline``
+       - Returns an aggregation pipeline with one pipeline
+         stage per line.
+
+     * - With the :option:`--explain <mongotranslate --explain>` option 
+       - ``--format none``
+       - Returns all fields on one line.
+
+     * - With the :option:`--explain <mongotranslate --explain>` option
+       - ``--format multiline``
+       - Returns one field per line, with additional formatting for
+         arrays and object subfields.
+---
+program: mongotranslate
+name: explain
+directive: option
+description: |
+  *Optional.* Returns the :manual:`explain </reference/command/explain>`
```

Comment:
We shouldn't link to the `explain` docs here. BIC's `explain` is different from MongoDB `explain`.

Reply:
Done.